os_migrate/roles/import_subnets/tasks/main.yml

```diff
@@ -15,8 +15,8 @@
     path: "{{ os_migrate_data_dir }}/subnets.yml"
   register: read_networks
 
-- name: import subnets
```

Comment:
This fails the test now. We need to get back to failing on the full test suite to be able to debug this.